transport/server.cc

```diff
@@ -159,6 +159,12 @@ cql_server::cql_server(distributed<cql3::query_processor>& qp, auth::service& au
 {
     namespace sm = seastar::metrics;
 
+    sm::label cql_error_label("type");
+    std::unordered_map<exceptions::exception_code, sm::label_instance> error_label_instances;
```

Comment:
no need for that, just add everything.

Reply:
you mean get rid of the map and put there 19 label instances? It seems rather ugly imo
And also, if anything changes in the `exceptions` namespace, creating the instances like this is just one less place to be affected

Reply:
but in your code later you manually call them, so it does not save anything, if your code would have use a loop, I can understand why you do it.

Reply:
I'm more keen on putting creating the metric list for the `cql` group in the loop as well then. Will do that